function.cpp

```diff
@@ -19,3 +19,32 @@ Price updatePrice(double DOC_service, double DOC_distance, double PAC_weight, do
 
 	return new_price;
 };
+//find position with address
+void Free(int *p){
+	delete p;
+}
+void Copy(int *p, int *q, int oldSize){
+	for(int i = 0; i < oldSize-1; i++){
+		q[i] = p[i];
+	}
+}
+int* Realloc(int *p, int newSize){
+	int *q = new int[newSize];
+	Copy (p, q, oldSize);
+	Free(p);
+	return q;
+}
+int* findPositionOfForm_fromAdress_findAdress(PackageShippingForm List[], std:: string look_for_string){
+	int i;
+	int *p = new int [i];
+	int sizeOfP = 0;
+
+	for(i=0; i<sizeof(List)/4; i++){
```

Comment:
Chắc gì 1 phần tử đối tượng List có kích thước là 4?

Cậu test thử xem sizeof(List) có bằng kích thước toàn bộ mảng không. Nếu có, thì số phần tử mảng sẽ là sizeof(List) / sizeof(List[0])

Reply:
Uầy quá đỉnh luôn